Makefile

```diff
@@ -0,0 +1,37 @@
+OUT_DIR = ./out
+JS_FILES = $(wildcard ./src/javascript/**)
+STYLE_FILES = $(wildcard ./src/css/**)
+STATIC_FILES = $(wildcard ./src/static/**)
+
+all: install_dependencies build
+
+build: clean js style markup static
+
+install_dependencies: ./package.json ./package-lock.json
+	$(info ********** install dependencies **********)
+	npm ci
+
+js: $(JS_FILES)
+	$(info ********** make JavaScript files **********)
+	mkdir -p $(OUT_DIR)
+	cp $+ $(OUT_DIR)
+
+style: $(STYLE_FILES)
+	$(info ********** make css files **********)
+	mkdir -p $(OUT_DIR)
+	npx postcss $+ --base src/css --dir $(OUT_DIR)
```

Comment:
css muss noch gepurged werden. ist glaub in der `postcss.config` mit der `NODE_ENV` variable gesteuert